netplan/libnetplan.py

```diff
@@ -64,19 +180,22 @@ def _string_realloc_call_no_error(function):
             continue
 
         if code < 0:  # pragma: nocover
-            raise LibNetplanException("Unknown error: %d" % code)
+            raise LibNetplanException(message="Unknown error: %d" % code)
         elif code == 0:
             return None  # pragma: nocover as it's hard to trigger for now
         else:
             return buffer.value.decode('utf-8')
-    raise LibNetplanException('Halting due to string buffer size > 1M')  # pragma: nocover
+    raise LibNetplanException(message='Halting due to string buffer size > 1M')  # pragma: nocover
 
 
 def _checked_lib_call(fn, *args):
     err = ctypes.POINTER(_GError)()
```

Comment:
We should probably be using the `NetplanError` type and `netplan_error_*` API here instead.